inc/db_base.php

```diff
@@ -349,7 +349,7 @@ function rename_table($old_table, $new_table, $table_prefix=true);
 	 * @param string $table The table
 	 * @param array $replacements The replacements
```

Comment:
Doc comments need updating too to reflect the new parameters (applies to every class, not just this one).